user.css

```diff
@@ -57,3 +57,12 @@
 .desktoproutes-homepage-takeover-ad-hptoComponent-parentContainer{
   display: none !important;
 }
+
+/*Remove download buttons*/
+.x-downloadButton-DownloadButton{
+  display:none;
+}
+
+.main-contextMenu-disabled {
+  display: none !important;
+}
```

Comment:
That is not good since it removes other disabled elements too.
![image](https://user-images.githubusercontent.com/43648146/127743776-23274fa6-c643-4df6-96b3-da5e30f98118.png)

The report button, for example.

Reply:
Are there any other disabled buttons on this list other than Report? If not, this new commit should work.